akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -12,6 +12,49 @@ To use Akka Cluster, you must add the following dependency in your project:
   version="$akka.version$"
 }
 
+## When and where to use Akka Cluster
+
+An architectural choice you have to make is if you are going to use a microservices architecture or
+a traditional distributed application, also known as a distributed monolith.
+
+Microservices has many attractive properties, such as the independent nature of microservices allows for
+multiple smaller and more focused teams that can deliver new functionality more frequently and can
+respond quicker to business opportunities. Reactive Microservices should be isolated, autonomous, and have
+a single responsibility as identified by Jonas Bonér in the book
+[Reactive Microservices Architecture: Design Principles for Distributed Systems](https://info.lightbend.com/COLL-20XX-Reactive-Microservices-Architecture-RES-LP.html).
+
+In general we recommend against using Akka Cluster and actor messaging between _different_ services because that
+would result in a too tight code coupling between the services and difficulties deploying these independent of
+each other, which is one of the main reasons for using a microservices architecture.
+See the discussion on
+@scala[[Internal and External Communication](https://www.lagomframework.com/documentation/current/scala/InternalAndExternalCommunication.html)]
+@java[[Internal and External Communication](https://www.lagomframework.com/documentation/current/java/InternalAndExternalCommunication.html)]
+in the docs of the [Lagom Framework](https://www.lagomframework.com) (where each microservice is an Akka Cluster)
+for some background on this.
+
+Nodes of a single service (collectively called a cluster) require less decoupling. They share the same code and
+are managed together, as a set, by a single team or individual. For this reason, intra-service communication
+can take advantage of Akka Cluster and actor messaging, which is convenient to use and has great performance.
+
+Between different services [Akka HTTP](https://doc.akka.io/docs/akka-http/current) or
+[Akka gRPC](https://developer.lightbend.com/docs/akka-grpc/current/) can be used for synchronous (yet non-blocking)
+communication and [Akka Streams Kafka](https://developer.lightbend.com/docs/alpakka/current/) for asynchronous communication.
+[Alpakka](https://developer.lightbend.com/docs/alpakka/current/) provides many connectors for integration with
+external systems. All those communication mechanisms work well with streaming of messages with
+end-to-end back-pressure, and the synchronous communication tools can also be used for single request
+response interactions. It is also important to note that when using these tools both sides of the communication
+does not have to be implemented with Akka, or not even Java/Scala.
+
+We recommend the microservices architectural pattern and Akka Cluster should then be used inside a single service for
```

Comment:
Then, I think these last two paras are now covered sufficiently above and can be removed, unless you want to leave this sentence as a summary.